podcasts/AccountViewController+TableView.swift

```diff
@@ -172,6 +182,12 @@ extension AccountViewController: UITableViewDataSource, UITableViewDelegate {
         case .supporterContributions:
             let supporterVC = SupporterContributionsViewController()
             navigationController?.pushViewController(supporterVC, animated: true)
+        case .changeAvatar:
+            guard let email = headerViewModel.profile.email,
+                  let safariViewController = GravatarSafariViewController(destination: .avatarUpdate(email: email)) else { return }
+            safariViewController.modalPresentationStyle = .automatic
+            present(safariViewController, animated: true)
+            Analytics.track(.accountDetailsChangeAvatar)
```

Comment:
We need to update [this spreadsheet](https://docs.google.com/spreadsheets/d/107jqrutZhU0fVZJ19SBqxxVKbV2NWSdQC9MFYdLiAxc/edit?gid=0#gid=0) when adding new events. I went ahead and created the entry for this one.

Reply:
Thank you! 🙇